ext/java/org/msgpack/jruby/Packer.java

```diff
@@ -30,20 +35,139 @@ public IRubyObject allocate(Ruby runtime, RubyClass type) {
     }
   }
 
+  static class ExtRegistry {
+    private Ruby runtime;
+    public RubyHash hash;
+    public RubyHash cache;
```

Comment:
Actually, why are these even public? They aren't accessed (and should absolutely not be)